### PR TITLE
drivers.md: Add CeaStor CSI driver

### PR DESCRIPTION
**Pull request by Tjuvenile**

### CODE REVIEW

**Comment by xing-yang** [comment]
Please sign CLA.

**Comment by Tjuvenile** [comment]
> 请签署CLA。

请审查这个 PR。谢谢！

File: book/src/drivers.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Bigtera VirtualStor (block)](https://github.com/bigtera-ce/ceph-csi) | `csi.block.bigtera.com` | v0.3, v1.0.0, v1.1.0 | A Container Storage Interface (CSI) Driver for Bigtera VirtualStor block storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion
 [Bigtera VirtualStor (filesystem)](https://github.com/bigtera-ce/ceph-csi) | `csi.fs.bigtera.com` | v0.3, v1.0.0, v1.1.0 | A Container Storage Interface (CSI)  Driver for Bigtera VirtualStor filesystem | Persistent | Read/Write Multiple Pods | Yes | Expansion
 [BizFlyCloud Block Storage](https://github.com/bizflycloud/csi-bizflycloud) | `volume.csi.bizflycloud.vn` | v1.2 | A Container Storage Interface (CSI) Driver for BizFly Cloud block storage | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion
+[CeaStor Block Storage](https://github.com/CECTC/ceastor-csi-driver) | `ceastor.csi.com` | v1.0 | A repository for the NVMe-oF CSI Driver for CeaStor | Persistent and Ephemeral | Depends on the storage backend used | Yes |
```
Comment (xing-yang) [inline book/src/drivers.md:29]: 1. Looks like the document is in Chinese. Can you also provide a doc in English?
2. Also the 3rd column is for support CSI spec version, not CSI driver version.
3. Can you be more specific about the access mode support?

Comment (Tjuvenile) [reply book/src/drivers.md]: 1. Unable to provide a document in English.
2. Modified.
3. Modified.

Apologies for the previous misunderstanding. Please continue to review this MR. Thank you!

**Comment by xing-yang** [comment]
Thanks for the update, but I still don't see a user guide in English.

**Comment by Tjuvenile** [comment]
> Thanks for the update, but I still don't see a user guide in English.

The English guide has been uploaded. Please continue to review this MR. Thank you!

**Comment by Tjuvenile** [comment]
> > Thanks for the update, but I still don't see a user guide in English.
> 
> The English guide has been uploaded. Please continue to review this MR. Thank you!

@xing-yang

**Pull request merged**